docs/PUBLISHING.md

```diff
@@ -93,7 +93,7 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|snapshot>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -p <product name>
```

Comment:
Good catch. Should this document also be updated to cover publishing snapshot builds, or are we going to leave that **only** to the CI servers?